build.gradle

```diff
@@ -97,7 +98,7 @@ dependencies {
   testImplementation "org.hamcrest:hamcrest-library:2.2"
   testImplementation 'org.mockito:mockito-core:4.3.1'
   testImplementation 'org.mockito:mockito-junit-jupiter:4.3.1'
-  testImplementation 'net.javacrumbs.json-unit:json-unit:2.30.0'
+  testImplementation 'net.javacrumbs.json-unit:json-unit:$versions.jsonUnit'
```

Comment:
```suggestion
  testImplementation "net.javacrumbs.json-unit:json-unit:$versions.jsonUnit"
```

Reply:
Interpolation requires double quotes, which is why you're getting build failures.